app/routes.py

```diff
@@ -1,2 +1,284 @@
+@rentals_bp.route("/check-in", methods=["POST"], strict_slashes=False)
+def return_videos():
+    
+    request_body = request.get_json()
+    if "customer_id" not in request_body and "video_id" not in request_body:
+            return make_response(jsonify({"details": "Invalid data"}), 400)
+
+    if not is_int(request_body["customer_id"]):
+        return make_response(jsonify({"details": "Invalid data"}), 400)
+    
+    if not is_int(request_body["video_id"]):
+        return make_response(jsonify({"details": "Invalid data"}), 400)
+
+    video_id = request_body["video_id"]
+    video = Video.query.get(video_id) 
+    if not video:
+        return make_response(jsonify({"details": "Invalid data"}), 404) 
+
+    customer_id = request_body["customer_id"] 
+    customer = Customer.query.get(customer_id) 
+
+    if not customer:
+        return make_response(jsonify({"details": "Invalid data"}), 404) 
+        
+
+    rental = Rental.check_in(customer_id, video_id)
+    print(rental)
+    if rental == None:
+        return make_response(jsonify({"details": "Invalid data"}), 400)
+
+    return jsonify(rental.to_dict()), 200
+
+# GET RENTALS FOR SPECIFIC CUSTOMER +++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
+@customers_bp.route("/<customer_id>/rentals", methods=["GET"], strict_slashes=False)
+def custumers_check_outs(customer_id):
+
+    customer = Customer.query.get(customer_id)
+
+    if Customer.query.get(customer_id) is None:
+        return make_response({"details": "Invalid data"}, 404)
+
+    rentals = db.session.query(Rental)\
+        .join(Customer, Customer.customer_id==Rental.customer_id)\
+        .join(Video, Video.video_id==Rental.video_id)\
+        .filter(Customer.customer_id==customer_id)
+
+    videos_rent = []
+    for rental in rentals:
+        video = Video.query.get(rental.video_id)
+        videos_rent.append({
+            "release_date":video.release_date.date().isoformat(),
+            "title": video.title,
+            "due_date": rental.due_date.isoformat()
+        })
+    return jsonify(videos_rent), 200
+
+# GET RENTALS FOR SPECIFIC VIDEO_ID +++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
+@videos_bp.route("/<video_id>/rentals", methods=["GET"], strict_slashes=False)
+def get_customer_with_rental(video_id):
+    video = Video.query.get(video_id)
+
+    if video is None:
+        return make_response({"details": "Invalid data"}, 404)
+
+    rentals = Rental.query.filter(Rental.video_id == video_id).all()
+
+    customers = []
+    for rental in rentals:
+        customer = Customer.query.get(rental.customer_id)
+
+        if customer is None:
+            continue
+
+        customer = {
+            "due_date": rental.due_date.isoformat(),
+            "name": customer.name,
+            "phone": customer.phone,
+            "postal_code": customer.postal_code
+        }
+
+        customers.append(customer)
+    
+    return jsonify(customers), 200
```

Comment:
Routes look great! You can further refactor this code into 3 files for customer, rental, and video.